test/http_ex/backend/mock_test.exs

```diff
@@ -183,5 +183,124 @@ defmodule HTTPEx.Backend.MockTest do
                      Mock.verify!(self())
                    end
     end
+
+    test "min_calls not met" do
+      Mock.expect_request!(
+        endpoint: "http://www.example.com",
+        expect_body: "GET",
+        min_calls: 3,
+        response: %{status: 200, body: "OK"}
+      )
+
+      Mock.request(%Request{
+        client: :httpoison,
+        url: "http://www.example.com",
+        method: :get,
+        body: "GET"
+      })
+
+      Mock.request(%Request{
+        client: :httpoison,
+        url: "http://www.example.com",
+        method: :get,
+        body: "GET"
+      })
```

Comment:
The `client: :httpoison` and `body: "GET"` are not important.

You used `Enum.each` in one of the tests, let's use that in the others too to prevent repeating the same blocks.

Reply:
Ah right, it's `Request.init(...)` that sets the client, but that's not used in this test file.